pkpdapp/pkpdapp/urls.py

```diff
@@ -27,4 +27,12 @@
     path('generic/', views.GenericView.as_view(), name='generic'),
     path('simulate/', include('simulate.urls')),
     path('django_plotly_dash/', include('django_plotly_dash.urls')),
+    path('dataset/<int:pk>/',
```

Comment:
Does it probably make sense to have separate apps for the dataset, pkpd_models and projects? At the moment there isn't much functionality but I am expected this to change within the next couple PRs. What do you think?

Reply:
I'm not sure, I've never done a large django app so I'm unclear on the delineation between apps. My feeling at the moment is that we should have one "main" app, which is the pkpd app, and then other "child" apps that can use the "pkpd" app. Ideally these "child" apps are independent from each other. Datasets, pkdpd_models and projects are quite interrelated, so might be better in the main "pkpd" app? 

But as I said, I'm not sure, so perhaps after a few more PRs it will be clear they need to be separated....?

Reply:
Sounds good. I guess we can just see. :)
I would argue though that the model and data are somewhat independent, and the project brings them together. I could see that maybe the project remains in the pkpdapp?